src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/dial_number_request.cc

```diff
@@ -130,7 +130,9 @@ void DialNumberRequest::on_event(const event_engine::Event& event) {
 
   const bool is_success = mobile_apis::Result::SUCCESS == result_code;
   const bool is_info_valid =
-      message[strings::msg_params].keyExists(strings::info);
+      message[strings::msg_params].keyExists(strings::info) && 
+      smart_objects::SmartType_String ==
+          message[strings::msg_params][strings::info].getType();
```

Comment:
@ValeriiMalkov It looks like a local fix. It checks type of info only for DialNumberRequest, if for instance the wrong type of info comes to another requester the crash can repeat. I think we need to validate info type before the specific response created.

Reply:
Agree we should check the type in info parameter while validating smart object

Reply:
@LuxoftAKutsan @ZhdanovP Please see fix in second commit 9cd0665c0fd1efad238adbd8214ef572dbe5af1d